### PR TITLE
James transportation begins

### PR DESCRIPTION
**Pull request by JamesBusch**

### CODE REVIEW

File: include/transportation.hh [hunk 1]
```diff
@@ -12,27 +12,71 @@
...
+#include <random>
 
-using namespace std;
+#include "postalCodeHash.hh"
+#include "agent.hh"
 
-enum TransportType { BUS, CAR, WALK };
+using namespace std;
 
 //Forward declarations
```
Comment (danyal1234) [inline include/transportation.hh:23]: remove all these

File: include/transportation.hh [hunk 2]
```diff
@@ -12,27 +12,71 @@
...
     private:
-    TransportType type;
-    Location* destination;
-    bool isStillInTransport;
+    PostalCodeHash *postalCodes;
+    std::vector<Location> locationList;
+
+    int randomInRange(int floor, int ceiling);
 
-    public:
```
Comment (danyal1234) [inline include/transportation.hh:31]: ur prob gonna need a public function called on timestep, but thats prob it? we can decide later

File: src/geographicalrisk.cpp [hunk 3]
```diff
@@ -34,6 +34,8 @@ void GeographicalRisk::updateAvgCountsAndRisk() {
         socialDistancing = (10.0 - (double)socialDistancingSeverity) / 10.0;
     }
 
+    cout << socialDistancing << endl;//TODO REMOVE THIS AND ADD IN GET SOCIAL DISTANCING METHOD
```
Comment (danyal1234) [inline src/geographicalrisk.cpp:37]: what this mean?

Comment (JamesBusch) [reply src/geographicalrisk.cpp:37]: This removes one of the warnings

File: include/transportation.hh (hunk 2 above)
Comment (JamesBusch) [reply include/transportation.hh:31]: will be adding later

File: src/geographicalrisk.cpp (hunk 3 above)
Comment (danyal1234) [reply src/geographicalrisk.cpp:37]: weird, ok for now

**Pull request merged**